org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/repl/WollokRepl.xtend

```diff
@@ -113,6 +113,10 @@ class WollokRepl {
 		println(obj?.toString.returnStyle)
 	}
 
+	def dispatch doPrintReturnValue(WollokObject wo) {
+		println(wo?.call("printString").toString.returnStyle)
+	}
+
 	def dispatch doPrintReturnValue(String obj) {
```

Comment:
I do not understand why is this necessary, can Strings be received here? Shouldn't we receive always a wollok object?